dcs/settings.py

```diff
@@ -41,10 +38,12 @@
     'django.contrib.staticfiles',
 
     # External Application
+    'django_extensions',
 
 
     # Internal Application
-    'catalog',
+    'crew',
+    'multimedia',
```

Comment:
`crew` is the apps that take care about multimedia's crew
` multimedia` (previously `catalog`) is the multimedia itself.